practical-react-unit-testing-with-jest-mocks-and-spies/README.md

```diff
@@ -2,38 +2,38 @@
 
 **Exercises**
 
-Manual Mocks and Bypassing mocks
+Manual Mocking
 
-[Exercise](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/practical-react-unit-testing-with-jest-mocks-and-spies/Exercise1/problem?file=src/toSelectOptions.test.tsx)
+[Exercise](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/practical-react-unit-testing-with-jest-mocks-and-spies/Exercise1/problem?file=src/Select.test.tsx)
 
-[Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/practical-react-unit-testing-with-jest-mocks-and-spies/Exercise1/solution?file=src/toSelectOptions.test.tsx)
+[Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/practical-react-unit-testing-with-jest-mocks-and-spies/Exercise1/solution?file=src/Select.test.tsx)
 
-Mocking API calls
+Mocking API calls and fetch
 
 [Exercise](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/practical-react-unit-testing-with-jest-mocks-and-spies/Exercise2/problem?file=/src/endpoints/get-data.test.ts)
 
 [Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/practical-react-unit-testing-with-jest-mocks-and-spies/Exercise2/solution?file=/src/endpoints/get-data.test.ts)
 
-Automatic Mocking
+Mocking with mock files
 
-[Exercise](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/practical-react-unit-testing-with-jest-mocks-and-spies/Exercise3/problem?file=/src/Select.test.tsx)
+[Exercise](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/practical-react-unit-testing-with-jest-mocks-and-spies/Exercise3/problem?file=/src/getDefaultSelectOptions.tsx)
 
-[Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/practical-react-unit-testing-with-jest-mocks-and-spies/Exercise3/solution?file=/src/Select.test.tsx)
+[Solution](https://codesandbox.io/p/sandbox/github/bitovi/trainings/tree/main/practical-react-unit-testing-with-jest-mocks-and-spies/Exercise3/solution?file=/src/getDefaultSelectOptions.test.tsx)
```

Comment:
The `file` is now pointing to something that doesn’t exist in this exercise.

Reply:
All of these links are pointing at the `main` branch and may not be valid until this PR is merged, especially in this case where a file has been added.

Reply:
I copied the links, updated the branch to be for this PR, and put them into this comment: https://github.com/bitovi/trainings/pull/82#issuecomment-1808652488